mne_sandbox/connectivity/cfc.py

```diff
@@ -0,0 +1,506 @@
+        between n_pairs of channels. Indices correspond to rows of `data`.
+    pac_func : string, ['plv', 'glm', 'mi_canolty', 'mi_tort', 'ozkurt']
+        The function for estimating PAC. Corresponds to functions in pacpy.pac
+    ev : array-like, shape (n_events,) | None
+        Indices for events. To be supplied if data is 2D and output should be
+        split by events. In this case, tmin and tmax must be provided
+    ev_grouping : array-like, shape (n_events,) | None
+        Calculate PAC in each group separately, the output will then be of
+        length unique(ev)
+    tmin : float | None
+        If ev is not provided, it is the start time to use in inst. If ev
+        is provided, it is the time (in seconds) to include before each
+        event index.
+    tmax : float | None
+        If ev is not provided, it is the stop time to use in inst. If ev
+        is provided, it is the time (in seconds) to include after each
+        event index.
+    baseline : array, shape (2,) | None
+        If ev is provided, it is the min/max time (in seconds) to include in
+        the amplitude baseline. If None, no baseline is applied.
+    baseline_kind : str
+        What kind of baseline to use. See mne.baseline.rescale for options.
+    scale_amp_func : None | function
+        If not None, will be called on each amplitude signal in order to scale
+        the values. Function must accept an N-D input and will operate on the
+        last dimension. E.g., skl.preprocessing.scale
+    use_times : array, shape (2,) | None
+        If ev is provided, it is the min/max time (in seconds) to include in
+        the PAC analysis. If None, the whole window (tmin to tmax) is used.
+    npad : int | 'auto'
+        The amount to pad each signal by before calculating phase/amplitude if
+        the input signal is type Raw. If 'auto' the signal will be padded to
+        the next power of 2 in length.
+    return_data : bool
+        If True, return the phase and amplitude data along with the PAC values.
+    concat_epochs : bool
+        If True, epochs will be concatenated before calculating PAC values. If
+        epochs are relatively short, this is a good idea in order to improve
+        stability of the PAC metric.
+    n_jobs : int
+        Number of CPUs to use in the computation.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    pac_out : array, dtype float, shape (n_pairs, [n_events])
+        The computed phase-amplitude coupling between each pair of data sources
+        given in ixs.
+    """
+    from ..externals.pacpy import pac as ppac
+    if pac_func not in _pac_funcs:
+        raise ValueError("PAC function {0} is not supported".format(pac_func))
+    func = getattr(ppac, pac_func)
+    ixs = np.array(ixs, ndmin=2)
+
+    if data.ndim != 2:
+        raise ValueError('Data must be shape (n_channels, n_times)')
```

Comment:
generally when there's a wrong param we indicate its value or type to be a bit more explicit